### PR TITLE
Take care of issues #610(partly), #642, #643

### PR DESCRIPTION
**Pull request by Schuermi7**
There where 3 issues, which needed a little effort to clean up in the documentation.
The NXLog issue was tested sucessfully with /var/spool/nxlog.  
I suggest, keeping the ``chown`` section, assuring that the file is owned by nxlog but changing the path to the correct version now.
One minor typo was fixed.

### CODE REVIEW

**Comment by jalogisch** [comment]
May I ask why you closed this? I am on the way review that and does you want to apply a bigger/other PR with this changes?

**Comment by Schuermi7** [comment]
I wanted to rebase on the newest branch and deleted the PR by accident. I am going to restore it.

File: pages/gelf.rst [hunk 1]
```diff
@@ -50,6 +50,8 @@ Prepend the following structure to your GELF message to make it chunked:
 
 All chunks **MUST** arrive within 5 seconds or the server will discard all already arrived and still arriving chunks. A message **MUST NOT** consist of more than 128 chunks.
 
+Please note, that the UDP-Inputs of Graylog use the ``SO_REUSEPORT`` socket option, which was introduced in Linux kernel version 3.9.
```
Comment (jalogisch) [inline pages/gelf.rst]: I would personal made a NOTICE block out of this. Just to make it more visible.

**Review by jalogisch — COMMENTED** [review]
Thank you for this work on the documentation. I have just a small note - what do you think about that?

**Comment by Schuermi7** [comment]
I added an attention block because I think, that this is a pretty good to know in commercial environments and it is not only worth noticing but worth to get your attention. It was 2 Commits, because at first i didn't understand, how your Boxing-System works.

**Review by jalogisch — APPROVED** [review]
LGTM thanks works perfect!

**Pull request merged**